dockstore-webservice/src/main/templates/io/dockstore/webservice/resources/Description.java

```diff
@@ -82,12 +70,12 @@ public class Description implements ReaderListener {
     public static final String TOS_LOCATION = "https://github.com/dockstore/dockstore-ui2/raw/develop/src/assets/docs/Dockstore_Terms_of_Service.pdf";
 
     @Override
-    public void beforeScan(Reader reader, Swagger swagger) {
-
+    public void beforeScan(OpenApiReader reader, OpenAPI openAPI) {
+        // intentionally left empty
     }
 
     @Override
-    public void afterScan(Reader reader, Swagger swagger) {
-        swagger.addSecurityDefinition(JWT_SECURITY_DEFINITION_NAME, new ApiKeyAuthDefinition("Authorization", In.HEADER));
+    public void afterScan(OpenApiReader reader, OpenAPI openAPI) {
+        // openAPI.addSecurityItem(new SecurityRequirement(JWT_SECURITY_DEFINITION_NAME, new ApiKeyAuthDefinition("Authorization", In.HEADER)));
```

Comment:
Unintentionally commented out?

Reply:
Took a look, looks like this workaround was replaced with the annotation 
`@SecuritySchemes({@SecurityScheme(type = SecuritySchemeType.HTTP, name = JWT_SECURITY_DEFINITION_NAME, scheme = "bearer")})` that appears on our resources and results in 

https://github.com/dockstore/dockstore/blob/dropwizarrd_update4/dockstore-webservice/src/main/resources/openapi3/openapi.yaml#L10845-L10848